graftm/create.py

```diff
@@ -198,6 +203,44 @@ def cleanup(self, the_trashcan):
             else:
                 os.remove(every_piece_of_junk)
     
+    def _define_range(self, sequences):
+        '''
+        define_range - define the maximum range within which two hits in a db 
+        search can be linked. This is defined as 1.5X the average length of all
+        reads in the database. 
+        NOTE: in the interest of speed this function assumes that the FASTA 
+        input is correctly formated. (i.e. headers start with '>' and sequences
+        do not. Nothing else is present within the file).
+        
+        Parameters
+        ----------
+        sequences : str
+            A path to the sequences in FASTA format. This fasta file is assumed 
+            to be in the correct format. i.e. headers start with '>'
+            
+        Returns
+        -------
+        max_range : int 
+            As described above, 1.5X the size of the average length of genes 
+            within the database
+        '''
+        sequence_count = 0
+        total_sequence = 0
+        
+        cmd = 'cat %s' % sequences # output the sequences to STDOUT
```

Comment:
I think it would be better to just use a normal fasta format reader here. Writing one from scratch may lead to problems. Is it really that much faster?